blocks/components/editable/index.js

```diff
@@ -233,7 +233,7 @@ export default class Editable extends wp.element.Component {
 		}
 
 		content = wp.element.renderToString( content );
-		this.editor.setContent( content );
+		this.editor.setContent( content, { format: 'raw' } );
```

Comment:
Yeah, I can see how we might want some sort of element validation, but I don't think it'll be in the direction of...

1. Try setting invalid HTML to Editable
2. TinyMCE cleans it up
3. TinyMCE passes back clean HTML to us

...but rather more like...

1. Editor state is always valid
2. User manages to enter invalid markup into Editable
3. It's cleaned up when markup is converted back into an element before being updated in state ([related](https://github.com/WordPress/gutenberg/blob/6f5deecbe8146a65038f190a656f8af265d451b5/blocks/components/editable/index.js#L46-L60), #523)